'replacement transaction underpriced' in test_mumbai

### Describe the bug

In both tests of test_mumbai.py, it fails with 'replacement transaction underpriced'. 

Examples seen in [this log](https://github.com/oceanprotocol/ocean.py/actions/runs/3528621685/jobs/5918918059). 

#1026 discussed this problem a lot, but focused more on documenting the solution. It got completed & closed earlier today. But it seems we need to also do more. brownie makes it easy to increase the amount of gas paid, [**see here**](https://eth-brownie.readthedocs.io/en/stable/core-gas.html). That would be the trick here. 

#1064 encountered this issue too, for Goerli. It was closed as duplicate with #1026. Regardless, let's get a better solution to make testing more robust.

### Full traceback

Here's the traceback for test_mumbai::test_nonocean_tx:

```text
_______________________________ test_nonocean_tx _______________________________

tmp_path = PosixPath('/tmp/pytest-of-runner/pytest-0/test_nonocean_tx0')

    def test_nonocean_tx(tmp_path):
        """Do a simple non-Ocean tx on Mumbai. Only use Ocean config"""
        # setup
        connect_to_network("mumbai")
        config = _remote_config_mumbai(tmp_path)
        ocean = Ocean(config)
        accounts.clear()
        (alice_wallet, bob_wallet) = get_wallets(ocean)
    
        # Simplest possible tx: Alice send Bob some fake MATIC
        bob_eth_before = accounts.at(bob_wallet.address).balance()
    
        normalized_unixtime = time.time() / 1e9
        amt_send = 1e-8 * (random.random() + normalized_unixtime)
    
        print("Do a send-Ether tx...")
        try:  # it can get away with "insufficient funds" errors, but not others
            alice_wallet.transfer(bob_wallet.address, f"{amt_send:.15f} ether")
        except ValueError as error:
            if "insufficient funds" in str(error):
                warnings.warn(UserWarning("Warning: Insufficient test MATIC"))
                return
>           raise (error)

tests/integration/remote/test_mumbai.py:39: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/integration/remote/test_mumbai.py:34: in test_nonocean_tx
    alice_wallet.transfer(bob_wallet.address, f"{amt_send:.15f} ether")
/opt/hostedtoolcache/Python/3.8.14/x[64](https://github.com/oceanprotocol/ocean.py/actions/runs/3528621685/jobs/5918918059#step:7:65)/lib/python3.8/site-packages/brownie/network/account.py:644: in transfer
    receipt, exc = self._make_transaction(
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/brownie/network/account.py:752: in _make_transaction
    exc = VirtualMachineError(e)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = VirtualMachineError(ValueError({'code': -32000, 'message': 'replacement transaction underpriced'}))
exc = {'code': -32000, 'message': 'replacement transaction underpriced'}

    def __init__(self, exc: ValueError) -> None:
        self.txid: str = ""
        self.source: str = ""
        self.revert_type: str = ""
        self.pc: Optional[int] = None
        self.revert_msg: Optional[str] = None
        self.dev_revert_msg: Optional[str] = None
    
        try:
            exc = exc.args[0]
        except Exception:
            pass
    
        if not (isinstance(exc, dict) and "message" in exc):
            raise ValueError(str(exc)) from None
    
        if "data" not in exc:
>           raise ValueError(exc["message"]) from None
E           ValueError: replacement transaction underpriced

/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/brownie/exceptions.py:96: ValueError
----------------------------- Captured stdout call -----------------------------
alice_wallet.address = '0x8bE6c8126C4383745c25F3b3Ca76D73E3Caf8544'
bob_wallet.address = '0xD[67](https://github.com/oceanprotocol/ocean.py/actions/runs/3528621685/jobs/5918918059#step:7:68)64a9F3387B14[69](https://github.com/oceanprotocol/ocean.py/actions/runs/3528621685/jobs/5918918059#step:7:70)2CC783BcF5604a23EBb[77](https://github.com/oceanprotocol/ocean.py/actions/runs/3528621685/jobs/5918918059#step:7:78)9f'
Do a send-Ether tx...
```

--- a/tests/integration/remote/test_mumbai.py
+++ b/tests/integration/remote/test_mumbai.py
@@ -7,6 +7,8 @@ import time
 import warnings
 
 from brownie.network import accounts
+from brownie.network import priority_fee, gas_price
+from brownie.network.gas.strategies import GasNowScalingStrategy
 
 from ocean_lib.ocean.ocean import Ocean
 from ocean_lib.web3_internal.utils import connect_to_network
@@ -18,6 +20,8 @@ def test_nonocean_tx(tmp_path):
     """Do a simple non-Ocean tx on Mumbai. Only use Ocean config"""
     # setup
     connect_to_network("mumbai")
+    _set_aggressive_gas_fees()
+
     config = _remote_config_mumbai(tmp_path)
     ocean = Ocean(config)
     accounts.clear()
@@ -46,8 +50,11 @@ def test_ocean_tx__create_data_nft(tmp_path):
     """On Mumbai, do a simple Ocean tx: create_data_nft"""
     # setup
     connect_to_network("mumbai")
+    _set_aggressive_gas_fees()
+
     config = _remote_config_mumbai(tmp_path)
     ocean = Ocean(config)
+
     accounts.clear()
     (alice_wallet, _) = get_wallets(ocean)
 
@@ -77,3 +84,10 @@ def _remote_config_mumbai(tmp_path):
     }
 
     return config
+
+
+def _set_aggressive_gas_fees():
+    # Mumbai uses EIP-1559. So, dynamically determine priority fee & gas price
+    priority_fee("auto")
+    gas_strategy = GasNowScalingStrategy("fast", increment=2.0)
+    gas_price(gas_strategy)

--- a/tests/integration/remote/test_mumbai.py
+++ b/tests/integration/remote/test_mumbai.py
@@ -7,8 +7,7 @@ import time
 import warnings
 
 from brownie.network import accounts
-from brownie.network import priority_fee, gas_price
-from brownie.network.gas.strategies import GasNowScalingStrategy
+from brownie.network import priority_fee
 
 from ocean_lib.ocean.ocean import Ocean
 from ocean_lib.web3_internal.utils import connect_to_network
@@ -87,7 +86,5 @@ def _remote_config_mumbai(tmp_path):
 
 
 def _set_aggressive_gas_fees():
-    # Mumbai uses EIP-1559. So, dynamically determine priority fee & gas price
+    # Mumbai uses EIP-1559. So, dynamically determine priority fee
     priority_fee("auto")
-    gas_strategy = GasNowScalingStrategy("fast", increment=2.0)
-    gas_price(gas_strategy)

